packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -16,9 +19,7 @@ const propTypes = {
   /**
    * An array of objects to be displayed as quick link options.
    */
-  quickLinks: PropTypes.arrayOf(PropTypes.shape({
-    text: PropTypes.string.isRequired,
-  })),
```

Comment:
i added just the text validation here because text is used below for the key. The idea is if a component takes props and uses some value off them, it should declare the type for that prop's value.